WordPress/Classes/ViewRelated/Blog/Blog Details/Detail Header/BlogDetailHeaderView.swift

```diff
@@ -53,16 +55,11 @@ class BlogDetailHeaderView: UIView {
     }
 
     @objc func refreshIconImage() {
-        if let blog = blog,
-            blog.hasIcon == true {
-            titleView.siteIconView.imageView.downloadSiteIcon(for: blog)
-        } else if let blog = blog,
-            blog.isWPForTeams() {
-            titleView.siteIconView.imageView.tintColor = UIColor.listIcon
```

Comment:
This is how it used to look:

<img width="320" alt="Screenshot 2024-07-23 at 5 54 33 PM" src="https://github.com/user-attachments/assets/19d69798-1864-4433-a0fe-847f1ee14f92">

Is it worth keeping? We now have the new placeholders that show the first letter of the site name.

Reply:
I don't see a huge difference in keeping or removing this special placeholder icon 🙂